### PR TITLE
Css animations

### PR DESCRIPTION
**Pull request by AmerMekic**
# Description

Added some css changes to our website: 

- Color gradient to the MovieCard background
- MovieCard height and width reduced so it fits a little better
- MovieCard translateX animation on entering the website
- MovieCard hover scale animation
- MovieCard box-shadow
- NavBar box-shadow
- NavBar hide and show animation during scroll
- Active link (home, favorites, login) is now colored blue if you're on the corresponding link
- Categories on hover scale and change color animation
- Login box gradient color
- Login box box-shadow
- Login box appear animation on entering login link

Please for all animations see the deploy preview on Netlify

## Screenshots

login-look

home-page-look

categories-on-hover

## Type of change

Please delete options that are not relevant.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/Pages/Login.js [hunk 1]
```diff
@@ -50,15 +52,13 @@ const Login = () => {
         value={email}
         onChange={handleChange}
         onKeyPress={handleKeyPress}
-        variant="outlined"
+        variant="filled"
         sx={{width: '60%', '& .MuiInputLabel-root': {color: 'white'}, '& fieldset': {borderColor: 'white'}, '& input': {color: 'white'}}}
       />
-      <Button variant="contained" size="medium" sx={{width: '60%', marginTop: '10px'}} onClick={handleClick}>
+      <Button variant="contained" size="medium" sx={{width: '60%', marginTop: '30px'}} onClick={handleClick}>
```
Comment (zlayabekrija) [inline src/Pages/Login.js]: Please avoid using inline styles in the future, please use styled components for that purpose.

File: src/components/MovieList.js [hunk 2]
```diff
@@ -87,17 +87,20 @@ const MovieList = (props) => {
...
-            movieGenre={getGenres(movie.genres)}
-            moviePoster={movie.posterUrl}
-            movieActors={movie.actors}
-            movieDescription={movie.plot}
-            movieFavorite={isFavorite(movie.title)}
-            handleFavorites={handleAddToFavorites}
-          />
+        {movies?.map((movie, index) => (
+          //handles the slow loading of a list
+          <div style={{animationDelay: index < 24 ? index * 0.02 + 's' : 0.02}} className="slow-load">
```
Comment (zlayabekrija) [inline src/components/MovieList.js]: Again we could we utilize, styled componets for this purpose and pass the index vale as a prop.

```
const StyledDiv=styled(div)`
animationDelay: ${props=> props.index < 24 ? props.index * 0.02 : 0.02}s}
` 
```

**Review by zlayabekrija — APPROVED** [review]
As usual, great work @AmerMekic, you should be proud of yourself.
Please address comments given in future pr's.

It's big thumbs up from me.

File: src/components/MovieList.js (hunk 2 above)
Comment (AmerMekic) [reply src/components/MovieList.js]: I will try to make this in the next css pr

File: src/Pages/Login.js (hunk 1 above)
Comment (AmerMekic) [reply src/Pages/Login.js]: I will try and make it in the next pr

**Comment by AmerMekic** [comment]
Thank you! @zlayabekrija

**Review by RejzovicEmir — APPROVED** [review]

**Pull request merged**